README.md

```diff
@@ -1 +1,184 @@
+
+- **This program has used HTML as markup language**
+- **This program has used CSS for styling**- 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://anilshah042022.github.io/hello-world/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Google Chrome or any other browse installed in your computer. 
+
+Git installed in your computer
+
+### Setup
+
+Clone this repository to your desired folder:
+To clone this repository to your local computer, follow these dteps.
+  - Goto to location where you want to clone this repository.
+  - Right click anywhere on screen and select "git bash" here
+  - Give gollowing command in git bash
+  - git clone -b index-html-style-css-branch https://github.com/anilshah042022/hello-world.git
+
+
+### Install
+
+No installation is required to run this program except those mentioned in prerequisites.
+
+### Usage
+
+To run the project, go to the location where you have cloned gitHub repositoty
+and double click index.html file
+
+### Expected output
+You will see following output in Blue color
+
+Hello Microverse
+
+### Run tests
+
+No test runs are required
+
+### Deployment
+
+No deployment is required
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Anilkumar Shah**
+
+- GitHub: [@githubhandle](https://github.com/anilshah042022))
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **About Microverse Program**
+- [ ] **How to enroll in Microverse Program**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project consider giving a star to this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Student success team, My mentor, My friend Mr. Yonas Lemma and all my morning sessions members, stand-up session members and
+cohort-19 members for replying my querries
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [GNU General Public](./LICENSE) licensed.
+
+_NOTE: The GNU General Public License is a free, copyleft license for software and other kinds of works.
+
```

Comment:
- [ ] When I click the license Link it leads to a 404 page not found. It is recommended that we use MIT license instead because the MIT license gives express permission for users to reuse code for any purpose, sometimes even if code is part of proprietary software. Kindly fix this.

![image](https://user-images.githubusercontent.com/84156714/226914179-4131b606-2704-4284-911e-058920f29fde.png)

- In your root path add a file LICENSE.md and add the recommended license text from [here](https://choosealicense.com/licenses/mit/#).

![image](https://user-images.githubusercontent.com/84156714/226914775-791dad11-0ea0-42a7-8f33-942deec011aa.png)